src/apprt/gtk/ui/1.5/ccw-paste.blp

```diff
@@ -10,22 +10,62 @@ Adw.AlertDialog clipboard_confirmation_window {
     cancel: _("Cancel") suggested,
     ok: _("Paste") destructive
   ]
-  
+
   default-response: "cancel";
   close-response: "cancel";
-  
-  extra-child: ScrolledWindow {
-    width-request: 500;
-    height-request: 250;
-
-    TextView text_view {
-      cursor-visible: false;
-      editable: false;
-      monospace: true;
-      top-margin: 8;
-      left-margin: 8;
-      bottom-margin: 8;
-      right-margin: 8;
+
+  extra-child: Overlay {
+    ScrolledWindow text_view_scroll {
+      width-request: 500;
+      height-request: 250;
+
+      TextView text_view {
+        cursor-visible: false;
+        editable: false;
+        monospace: true;
+        top-margin: 8;
+        left-margin: 8;
+        bottom-margin: 8;
+        right-margin: 8;
+
+        styles [
+          "clipboard-content-view"
+        ]
+      }
+    }
+
+    [overlay]
+    Button reveal_button {
+      visible: false;
+      halign: end;
+      valign: start;
+      margin-end: 12;
+      margin-top: 12;
+
+      styles [
+        "destructive-action",
+      ]
```

Comment:
I would not use suggested or destructive action styles for the overlay. Just the icon in regular colors will be fine. The dialog already has suggested and destructive coloring. Seems a little overwhelming